specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2024-09-01-preview/appconfiguration.json

```diff
@@ -0,0 +1,3168 @@
+          "description": "A filter used to match keys. Syntax reference:\nhttps://aka.ms/azconfig/docs/keyvaluefiltering. This is only applicable when the resourceType is Kv.",
+          "type": "string"
+        },
+        "label": {
+          "description": "A filter used to match labels. Syntax reference:\nhttps://aka.ms/azconfig/docs/keyvaluefiltering. This is only applicable when the resourceType is Kv.",
+          "type": "string"
+        },
+        "tags":{
+          "description": "Am array of tag filters used to match tags.",
+          "type": "array",
+          "items": {
+            "description": "A filter used to match key-values by their tags, specified in `tagName=tagValue` format. This is only applicable when the resourceType is Kv.",
+            "type": "string"
+          }
+        },
+        "name":{
+          "description": "The name of the snapshot. This is only applicable when the resourceType is Snapshot.",
+          "type": "string"
+        }
+      }
+    },
+    "SasTokenGenerateResult": {
+      "description": "The result of a request to generate a SAS token.",
+      "type": "object",
+      "properties": {
+        "resourceType": {
+          "description": "The data plane resource type that the SAS token is authorized to access.",
+          "type": "string",
+          "readOnly": true
+        },
+        "signedParameters": {
+          "description": "The parameters to define the data plane resource scope.",
+          "$ref": "#/definitions/SignedParameters",
+          "readOnly": true
+        },
+        "exp":{
+          "description": "The time that the SAS token expires in the Universal ISO 8601 DateTime format. Max allowed expiration is 1 year from the time of token creation.",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "cch":{
+          "description": "Time (in seconds) for which the data plane response may be cached by clients. App Config service sets the Cache-Control response header `max-age` to the value that's specified on the SAS token. See [rfc9111](https://www.rfc-editor.org/rfc/rfc9111#name-max-age-2) for more details.",
+          "type": "number"
+        },
+        "sasKind":{
+          "description": "The kind of the SAS token.",
+          "type": "string",
+          "readOnly": true
+        },
+        "tokenValue":{
+          "description": "The value of the SAS token.",
+          "type": "string",
+          "format": "password",
+          "x-ms-secret": true,
+          "readOnly": true
+        }
+      }
+    },
+    "ResetSasParameters": {
+      "description": "Parameters used for resetting SAS token.",
+      "required": [
+        "sasKind"
+      ],
+      "type": "object",
+      "properties": {
+        "sasKind":{
+          "description": "The kind of the SAS token.",
+          "type": "string",
+          "enum": [
+            "Primary",
+            "Secondary"
```

Comment:
I'm not aware of there being primary and secondary SAS tokens. Only primary and secondary signing keys.

Reply:
@avanigupta

Reply:
I found the design doc updates. Deleting this.